### DISCUSSION TITLE
Wormhole Transfer failed

### DISCUSSION THREAD

**Discussion by ghost** [wormhole-foundation/wormhole#273] [Problems / bugs] [answered]
<img width="526" alt="image" src="https://user-images.githubusercontent.com/34089163/127303150-a1d35d52-3590-4b45-ab75-8457eb97add6.png">

I cross my erc20 LUNA from Ethereum to Solana, but it's still failed. I tried maybe 4~5 times, no one success, please help me. thank you

My Ethereum Address: 0x53A0eF6Fb89a5614d5DBaD737d74C1037CCD1F8E
My Solana Address: X3v7t8fnfyxZsuTNkqaf5muWVPU1CpurZ6C1DcJDwLx

**Comment by hendrikhofstadt** [comment, marked as answer]
This looks like a UI bug to me. The UI was built by a third-party: https://github.com/solana-labs/oyster
